docs/about/infrastructure/aws/how_to_use.md

```diff
@@ -0,0 +1,69 @@
+# How to use AWS in our projects
+
+## Getting credentials
+
+We try to use the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege). 
```

Comment:
The only additional note here is that cloudknot requires pretty expansive privileges. I think it's still the least privilege necessary, but it's a lot and one could easily burn a lot of money with cloudknot. I don't know if this point deserves a place in this document or not. Maybe this is just something you tell to people one-on-one when you grant them those kinds of permissions.